js/everything.js

```diff
@@ -1,3 +1,18 @@
+// DarkTheme Function:
+
+var modeIcon = document.getElementById("modeIcon");
+
+modeIcon.onclick = function(){
```

Comment:
We should have a a space before each `{`, after `if`, & after `else`.

Please run `npm test` locally to see all such issues.